packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -438,7 +440,16 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
         child: AutocompleteHighlightedOption(
           highlightIndexNotifier: _highlightedOptionIndex,
           child: Builder(
-            builder: (BuildContext context) => widget.optionsViewBuilder(context, _select, _options),
+            builder: (BuildContext context) {
+              return UnconstrainedBox(
+                constrainedAxis: Axis.vertical,
+                alignment: Alignment.topLeft,
```

Comment:
```suggestion
                alignment: followerAlignment,
```

This makes it vary properly with `optionsViewOpenDirection` and the ambient text direction.

(I think the vertical alignment doesn't affect anything because `constrainedAxis` means the heights should match anyway, so `optionsViewOpenDirection` would actually behave correctly anyway. RTL layout would be affected, though, whenever this alignment comes into play.)

Reply:
Good catch!